[2023-11-09T20:04:02.937+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mon_premier_dag.task_3 manual__2023-11-09T20:03:49.599147+00:00 [queued]>
[2023-11-09T20:04:02.952+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mon_premier_dag.task_3 manual__2023-11-09T20:03:49.599147+00:00 [queued]>
[2023-11-09T20:04:02.954+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-09T20:04:02.979+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): task_3> on 2023-11-09 20:03:49.599147+00:00
[2023-11-09T20:04:02.989+0000] {standard_task_runner.py:57} INFO - Started process 2225 to run task
[2023-11-09T20:04:02.994+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'mon_premier_dag', 'task_3', 'manual__2023-11-09T20:03:49.599147+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp2yqxn6vq']
[2023-11-09T20:04:03.000+0000] {standard_task_runner.py:85} INFO - Job 18: Subtask task_3
[2023-11-09T20:04:03.069+0000] {task_command.py:416} INFO - Running <TaskInstance: mon_premier_dag.task_3 manual__2023-11-09T20:03:49.599147+00:00 [running]> on host eaa713dd4990
[2023-11-09T20:04:03.184+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='votre_nom' AIRFLOW_CTX_DAG_ID='mon_premier_dag' AIRFLOW_CTX_TASK_ID='task_3' AIRFLOW_CTX_EXECUTION_DATE='2023-11-09T20:03:49.599147+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-09T20:03:49.599147+00:00'
[2023-11-09T20:04:03.187+0000] {logging_mixin.py:154} INFO - Exécution de la tâche 3
[2023-11-09T20:04:03.188+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-09T20:04:03.201+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=mon_premier_dag, task_id=task_3, execution_date=20231109T200349, start_date=20231109T200402, end_date=20231109T200403
[2023-11-09T20:04:03.247+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-09T20:04:03.270+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
